### PR TITLE
get all genres

### LINKED ISSUE #8: Implement /api/genres

### PR DESCRIPTION
**Pull request by jyotsnaravikumar**
## Type of PR

- [ ] Documentation changes
- [ X] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
/api/genres API implementation

## Issues Closed or Referenced

- Closes #8

### CODE REVIEW

**Comment by jofultz** [comment]
Is failing the checks expected?

File: src/main/java/com/microsoft/cse/helium/app/dao/GenresDao.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  private static String genreQuery =
+      "select m.genre from m where m.type = 'Genre' order by m.genre ";
+
+  public GenresDao(IConfigurationService configService) {
+    super(configService);
+  }
+
+  /**
+   * getActors.
```
Comment (jofultz) [inline src/main/java/com/microsoft/cse/helium/app/dao/GenresDao.java]: Mismatched comment

File: src/main/java/com/microsoft/cse/helium/app/dao/GenresDao.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+   *
+   * @return
+   */
+  public Mono<List<String>> getGenres() {
```
Comment (jofultz) [inline src/main/java/com/microsoft/cse/helium/app/dao/GenresDao.java]: Should we return Flux? I think Sanjeev may have used Flux for a long list in Actors. Let's make sure the return type matches for all of the get all type methods.

File: src/main/java/com/microsoft/cse/helium/app/dao/GenresDao.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    Mono<List<String>> selectedGenres =
+        this.context
+            .getBean(CosmosClient.class)
+            .getDatabase(this.cosmosDatabase)
+            .getContainer(this.cosmosContainer)
+            .queryItems(genreQuery, this.feedOptions)
+            .flatMap(
+                flatFeedResponse -> {
+                  return Flux.fromIterable(flatFeedResponse.results());
```
Comment (jofultz) [inline src/main/java/com/microsoft/cse/helium/app/dao/GenresDao.java]: Can we just return the cosmos response directly without flattening it out do we need the map and flatten?

**Comment by jyotsnaravikumar** [comment]
Closing this PR to recreate new one to run on Eddie's CI-CD changes

**Pull request closed without merging**